inc/item_ticket.class.php

```diff
@@ -685,6 +685,10 @@ static function dropdownMyDevices($userID = 0, $entity_restrict = -1, $itemtype
                 && Ticket::isPossibleToAssignType($itemtype)) {
                $itemtable = getTableForItemType($itemtype);
 
+               if (!$DB->fieldExists($itemtable, 'users_id')) {
```

Comment:
Is it normal to have an item without `users_id` field in `$CFG_GLPI["linkuser_types"]` ?
Maybe problem will be the same for `$CFG_GLPI["linkgroup_types"]`.

Reply:
Yes and no.

We have two options for adding items to ticket, adding current user's ones and add a global one.
In the example of simcard, i'm not sure, it's still valid for global part, but anyway, you're right, we should add users_id fields to table to permits also "mine" part (and respect the purpose of the function).